static/app/views/performance/traceDetails/traceViewDetailPanel.tsx

```diff
@@ -211,6 +212,34 @@ function EventDetails({detail, organization, location}: EventDetailProps) {
     .filter(isNotPerformanceScoreMeasurement)
     .sort();
 
+  const renderMeasurements = () =>  {
+      if(!detail.event)
+        {return null;}
```

Comment:
The formatting on this is kind of funky, can you run prettier on this file? Not sure why it didn't run but it seems like things also have 4 spaces for indentation here.